api/models/Campaign.js

```diff
@@ -12,6 +12,7 @@ const schema = new mongoose.Schema({
 
   // Properties cached from DS API.
   title: String,
+  run_id: Number,
```

Comment:
I'm definitely in favor of keeping our naming conventions consistent across platforms, but agree nothing to change today. We do use `currentCampaignRun` in [Phoenix](https://github.com/DoSomething/phoenix-js/blob/7b4c45f7fe31bab7023d2408ce92f89b06d570fa/lib/phoenix-campaign.js#L16) -- my hesitation with `run` is it feels a little too generic, or questionable if it's actual DS Campaign Run. how about `current_run` ?